### ISSUE TITLE
Integration for Elementor

### ISSUE THREAD

**Issue by mbautista** [altcha-org/wordpress-plugin#12]
Hello,
Thank you for this great plugin :)

I've tried to integrate with Elementor forms using the shortcode, but it does not seem to work :
- When inserting the shortcode in a form element it juste displays "This form requires javascript!"
- When inserting the shortcode outside the Elementor form it inject the altcha widget, but it does not seem to be effective, maybe because the widget is outside the form ?

So I created a new Elementor form field with the following code : 
```
/**
 * Altcha Elementor Integration
 */
if (is_plugin_active('elementor/elementor.php')){
  function altcha_register_new_form_fields( $form_fields_registrar ) {
    class Elementor_Altcha_Field extends \ElementorPro\Modules\Forms\Fields\Field_Base {
      public function get_type() {
        return 'altcha';
      }
      public function get_name() {
        return __('Altcha');
      }
      public function render( $item, $item_index, $form ) {
        echo do_shortcode('[altcha]');
      }
      public function validation( $field, $record, $ajax_handler ) {}
      public function update_controls( $widget ) {}
    }
    $form_fields_registrar->register( new Elementor_Altcha_Field() );
  }
  add_action( 'elementor_pro/forms/fields/register', 'altcha_register_new_form_fields' );
}
```

I have tested it and so far it seems to work. Could you please check if this is ok and if it's good for you integrate it to the next release ?

Thank you very much :)
Mathieu.

EDIT : Added `if (is_plugin_active('elementor/elementor.php')){...}`

**Comment by ovx** [comment]
Hi, thanks for the contribution, I'll add it to the plugin.

**Comment by ovx** [comment]
The full Elementor integration is implemented in the latest version 1.4.0 including the server verification.

Thanks again, and please report if you find any issues.
